collect_app/src/main/java/org/odk/collect/android/adapters/InstanceListCursorAdapter.java

```diff
@@ -57,26 +57,23 @@ public View getView(int position, View convertView, ViewGroup parent) {
         setUpSubtext(view);
 
         // Some form lists never contain disabled items; if so, we're done.
+        // Update: This only seems to be the case in Edit Saved Forms and it's not clear why...
         if (!shouldCheckDisabled) {
             return view;
         }
 
+        boolean formExists = true;
```

Comment:
This still could be false by default the you wouldn't need that else statement adding just formExists - true below:
`isFormEncrypted = base64RSAPublicKey != null && !base64RSAPublicKey.isEmpty();`

Reply:
Good point! I can clean that up.